portfolio/src/main/webapp/script.js

```diff
@@ -12,51 +12,46 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/*
-    The function showNext() allows the button to be clicked twice in order to
-    display a personal summary and a photograph. Because the second click is dependent
-    on the first, a global variable is required to determine when to show the picture.
-    Another roundabout solution would be to check whether the 'summaryCointainer'
-    is empty, eliminating the need for a global varible.
-*/
-var summaryVisble = false;
-var pictureVisbile =false;
+/** 
+ * Allows a button to be clicked successively in order to
+ * display a personal summary, then a photograph. 
+ */
+
 function showNext() {
-    if(summaryVisble == false){
-        showSummary();
-        summaryVisble = true;
-    } 
-    else if(pictureVisbile == false){ 
-        showPicture();
-        pictureVisbile = true;
+  const summaryContainer = document.getElementById('summary');
+  const imageContainer = document.getElementById('picture');
+  if (summaryContainer.innerText.length == 0) {
+    showSummary();
+    } else if (imageContainer.childElementCount == 0) { 
+      showPicture();
     }
 }
 
 function showSummary(){
-    const summaryContainer = document.getElementById('summary');
-    const summaryText = "Currently, I'm a 2nd year student"+ 
-    " at the Rose-Hulman Institute of Technology working towards a bachelor"+
-    " in Computer Science. I grew up in Louisville, Kentucky and moved on to"+
-    " college early.I completed two years at Western Kentucky University while"+
-    " simultaneously obtaining my highschool diploma. The summer of my graduation"+
-    " I was accepted as a Google CSSI student and spent 3 weeks at the Google Cambridge"+
-    " Office. A year later and I am now working as a virtual Google intern in my"+
-    " hometown of Louisville."
-    summaryContainer.innerText = summaryText;
+  const summaryContainer = document.getElementById('summary');
+  const summaryText = "Currently, I'm a 2nd year student"+ 
+  " at the Rose-Hulman Institute of Technology working towards a bachelor"+
+  " in Computer Science. I grew up in Louisville, Kentucky and moved on to"+
+  " college early.I completed two years at Western Kentucky University while"+
+  " simultaneously obtaining my highschool diploma. The summer of my graduation"+
+  " I was accepted as a Google CSSI student and spent 3 weeks at the Google Cambridge"+
+  " Office. A year later and I am now working as a virtual Google intern in my"+
+  " hometown of Louisville."
```

Comment:
Use single quotes: https://google.github.io/styleguide/jsguide.html#features-strings-use-single-quotes, and space before `+`.